go.mod

```diff
@@ -71,7 +95,7 @@ replace (
 )
 
 require (
-	dario.cat/mergo v1.0.0 // indirect
+	dario.cat/mergo v1.0.0
```

Comment:
Move this up to be with the direct deps and not down with the indirects. Typically this is done automatically unless this dep was changed by hand.